src/partials/hero.html

```diff
@@ -1,2 +1,91 @@
-<header class="header">
-</header>
\ No newline at end of file
+<div class="container hero-container">
```

Comment:
black only on your section and the entire length

<section class="hero"> --- background-color: var(--color-black);
  <div class="container hero-container">
    <h1 class="hero-title">Welcome to our collection of premium watches</h1>
    <div class="swiper_slider">